src/main/kotlin/org/jetbrains/kotlin/jupyter/config.kt

```diff
@@ -83,6 +83,26 @@ class RuntimeKernelProperties(val map: Map<String, String>): ReplRuntimeProperti
     }
 }
 
+data class KernelCfgFile(
```

Comment:
I think that name `KernelJupyterParams` fits better. Anyway, please rename this class to something without abbreviations

Reply:
Sounds like a good name, changed it.